src/shims/android/syscall.h

```diff
@@ -0,0 +1 @@
+#import <sys/syscall.h>
```

Comment:
please just have

```
#ifdef __ANDROID__
#include <sys/syscall.h>
#endif
```

in the right place instead of these overlaying of headers